### PR TITLE
add test performed to db model

### LINKED ISSUE #4940: Add LOINC column to the supported disease-device table
### Background
Our device table is a simplified version of the CDC's LIVD lookup table. One of the simplifications we've made is assuming that all tests performed on a device have the same LOINC code. It turns out this isn't true - a single device will have multiple "test performed" LOINC codes, usually one per disease tested. This hasn't been an issue in the past because we're not sending any results to states, but with the FHIR pilot we'll need to be more accurate about this test performed code.

The test performed code only matters to us during the ReportStream export. Therefore, rather than try to adjust our device model to support multiple LOINC codes (going from one-to-one to many-to-one) we'll simply track these LOINC codes separately, and reference them during the export. The simplest place to store the LOINCs is on our existing table for supported diseases/devices.

### Action Requested
Add a column to the "Device Supported Disease" table to store the test performed LOINC code. 

### Acceptance Criteria
The device/supported disease join table has an additional column added for test performed LOINC code. This column is available at both the database and model level.

Backfilling the data will be covered by a separate ticket, as will using the data during export.

### Additional Context
This work isn't directly FHIR related, but does need to be completed before we start sending flu/multiplex results to states. The test performed LOINC code is the only piece of data that most states use for identifying devices, so it's important for it to be accurate.

You can find the LIVD table here: https://www.cdc.gov/csels/dls/livd-codes.html#:~:text=Mapping%20tool%20for%20SARS%2DCoV%2D2

### PR DESCRIPTION
**Pull request by BobanL**
# BACKEND PULL REQUEST

## Related Issue

- Resolves #4940 

## Changes Proposed

- Adds the model changes for the db changes made here #4984

### CODE REVIEW

File: backend/src/main/java/gov/cdc/usds/simplereport/db/model/DeviceSupportedDisease.java [hunk 1]
```diff
@@ -23,4 +23,11 @@ public class DeviceSupportedDisease {
   @Id
   @Column(name = "supported_disease_id", nullable = false)
   private UUID supportedDiseaseId;
+
+  @Column private String testPerformedCode;
```
Comment (emmastephenson) [inline backend/src/main/java/gov/cdc/usds/simplereport/db/model/DeviceSupportedDisease.java:27]: This `testPerformedCode` will also need a setter at some point - up to you if you'd like to add that now or in a future PR.

**Review by emmastephenson — APPROVED** [review]
LGTM!

File: backend/src/main/java/gov/cdc/usds/simplereport/db/model/DeviceSupportedDisease.java (hunk 1 above)
Comment (BobanL) [reply backend/src/main/java/gov/cdc/usds/simplereport/db/model/DeviceSupportedDisease.java:27]: adding `@Setter `won't be hard to add when we need it in the future, so I'll leave it like this for now.

**Review by emyl3 — APPROVED** [review]
LGTM!!

**Pull request merged**